scraper.rb

```diff
@@ -0,0 +1,31 @@
+require 'pry'
+require 'require_all'
+require 'scraped'
+require 'scraped_page_archive/open-uri'
+
+require_rel 'lib'
+
+english_url = 'http://www.althingi.is/altext/cv/en/'
+icelandic_urls = {
+        members: 'http://www.althingi.is/thingmenn/althingismenn/',
+        alternates: 'http://www.althingi.is/thingmenn/thingmenn/varamenn-sem-sitja-a-althingi/',
+        substitues: 'http://www.althingi.is/thingmenn/thingmenn/varamenn-sem-hafa-tekid-saeti/'
+        }
```

Comment:
This is very odd formatting. As this is a new gem, you should really be making sure it passes `rubocop` checking…